src/main/java/com/google/collegeplanner/data/Class.java

```diff
@@ -0,0 +1,113 @@
+package com.google.collegeplanner.data;
+
+public class Class {
+  private String courseId;
+  private String name;
+  private String semester;
+  private int credits;
+  private String deptId;
+  private String description;
+  private String[] gradingMethod;
+  private String coreqs;
+  private String prereqs;
+  private String restrictions;
+  private String additionalInfo;
+  private String creditGrantedFor;
```

Comment:
For our purposes, do you think we need all of this information? (Should we be storing everything, or only information we need?)

Reply:
I was thinking we don't need to store all this information, but this was the information in Savion's datastore design doc.

Reply:
Personally I think it's all useful info especially for the algorithm. This also isn't everything - I left out some fields in my design doc that I didn't think were useful.